packages/react/src/themes/teams/components/List/listItemVariables.ts

```diff
@@ -21,6 +23,9 @@ export interface ListItemVariables {
 
 export default (siteVariables: any): ListItemVariables => {
   return {
+    minHeight: pxToRem(48),
+    rootPadding: `0 ${pxToRem(18)} 0 ${pxToRem(20)}`,
```

Comment:
can you add the new props `minHeight`, `rootPadding `, `contentMediaFontSize `, `contentMediaLineHeight ` to `ListItemVariables` interface?

Reply:
Sure, this was updated in some of the new PRs that got in..